### PR TITLE
docs: `integrations/providers` fixes

### PR DESCRIPTION
**Pull request by leo-gan**
Fixed documentation inconsistencies, formats. Added descriptions and links.
NOTE: I had to change several cookbooks and notebooks (unrelated to this PR) to pass the linking. Changes are small and easy. These changes were inflicted by `Updating ruff (... -> 0.4.2)`

### CODE REVIEW

**Comment by leo-gan** [comment]
@baskaryan @efriis @eyurtsev How to deal with errors in the `Vercel` check? I don't have access to the errors in this check hence cannot fix errors.
Does somebody from the LangChain team provide contributors with the error logs or fix them instead of contributors?

**Comment by sepiatone** [comment]
> @baskaryan @efriis @eyurtsev How to deal with errors in the `Vercel` check? I don't have access to the errors in this check hence cannot fix errors. Does somebody from the LangChain team provide contributors with the error logs or fix them instead of contributors?

I had the same [problem](https://github.com/langchain-ai/langchain/pull/20494). On trying to access the Vercel logs I always got a 404 error. (One of the maintainers fixed the problem, whatever it was.)

Is there some way we can have access in an automated way to the error logs? Then we can take responsibility for the correctness of our PRs and only depend on the maintainers for the actual review.

**Comment by baskaryan** [comment]
> > @baskaryan @efriis @eyurtsev How to deal with errors in the `Vercel` check? I don't have access to the errors in this check hence cannot fix errors. Does somebody from the LangChain team provide contributors with the error logs or fix them instead of contributors?
> 
> I had the same [problem](https://github.com/langchain-ai/langchain/pull/20494). On trying to access the Vercel logs I always got a 404 error. (One of the maintainers fixed the problem, whatever it was.)
> 
> Is there some way we can have access in an automated way to the error logs? Then we can take responsibility for the correctness of our PRs and only depend on the maintainers for the actual review.

it's a good question, @efriis and i should look into some way of propagating the vercel error logs to github

File: docs/docs/integrations/providers/nomic.ipynb
Comment (efriis) [inline docs/docs/integrations/providers/nomic.ipynb:1]: same comment - let's leave these as notebooks unless there's a strong reason to change them. Would prefer source of truth went the other direction instead.

Comment (leo-gan) [reply docs/docs/integrations/providers/nomic.ipynb:1]: @efriis In [this notebook](https://python.langchain.com/docs/integrations/providers/nomic/) there is only a single line of code
`from langchain_nomic import NomicEmbeddings`
I honestly don't understand what code are you talking about. It is all copied in [the new page](https://langchain-aybwsh2ji-langchain.vercel.app/docs/integrations/providers/nomic/) plus all missed integrations.


Are you talking about something else?

**Comment by hwchase17** [comment]
sorry no one reviewed earlier, now lots of merge conflicts

**Pull request closed without merging**